src/main/java/org/kiwiproject/dropwizard/error/config/CleanupConfig.java

```diff
@@ -24,12 +24,32 @@
     }
 
     /**
-     * The strategy to use for what to cleanup. Defaults to {@link CleanupStrategy#RESOLVED_ONLY}
+     * The strategy to use for what to cleanup. Defaults to {@link CleanupStrategy#ALL_ERRORS}
```

Comment:
```suggestion
     * The strategy to use for what to clean up. Defaults to {@link CleanupStrategy#ALL_ERRORS}.
```

According to IntelliJ's new grammar checking stuff:

> Did you mean the phrasal verb clean up instead of the noun 'cleanup'?
> 
> Incorrect:
> He promised to cleanup his room.
> 
> Correct:
> He promised to clean up his room.
> 

I also added a period to the end of the sentence...because sentences. :smirk: